trie/trie_test.go

```diff
@@ -842,15 +842,15 @@ func deleteString(trie *Trie, k string) {
 	trie.Delete([]byte(k))
 }
 
-func TestDecodeNode(t *testing.T) {
-	t.Parallel()
-	var (
-		hash  = make([]byte, 20)
-		elems = make([]byte, 20)
-	)
-	for i := 0; i < 5000000; i++ {
-		rand.Read(hash)
-		rand.Read(elems)
-		decodeNode(hash, elems)
-	}
-}
+//func TestDecodeNode(t *testing.T) {
+//	t.Parallel()
+//	var (
+//		hash  = make([]byte, 20)
+//		elems = make([]byte, 20)
+//	)
+//	for i := 0; i < 5000000; i++ {
+//		rand.Read(hash)
+//		rand.Read(elems)
+//		decodeNode(hash, elems)
+//	}
+//}
```

Comment:
Note this is commented, not sure what should be done with it though.

Reply:
Thanks! I'm trying to figure out how it ever worked, as `decodeNode(...)` will fail depending on the content of the provided input, and we're passing random input. Will circle back to this!